client/src/containers/projects/form/setup/restoration-project-detail/index.tsx

```diff
@@ -226,13 +226,13 @@ export default function RestorationProjectDetails() {
                 <div className="basis-1/2 space-y-2">
                   <FormLabel
                     tooltip={{
-                      title: "IPCC default value",
+                      title: "Tier 1 - IPCC Default value",
                       content: "TBD",
                     }}
                   >
-                    IPCC default value
+                    Tier 1 - IPCC Default value
```

Comment:
It might be better if we reference the name from [here](https://github.com/Vizzuality/tnc-blue-carbon-cost-tool/blob/f066990e8225382f1906c75dd37fbdf3a49b563a/shared/entities/carbon-inputs/sequestration-rate.entity.ts#L15)